src/migrations/migration.js

```diff
@@ -24,6 +25,32 @@ export default class Migration {
     return 'No description';
   }
 
+  get logger() {
+    if (this._logger) {
+      return this._logger;
+    }
+    return {
+      info:    console.info,
+      error:   console.error,
+      warning: console.warn
+    };
+  }
+
+  /**
+   * Tries to parse objectString to JSON Object, logs warning and returns {} on failure.
+   * @param {String} objectString
+   * @returns Parsed JSON Object or {} on failure
+   */
+  _parseJSON(objectString) {
```

Comment:
why _ (private) prefix I think we want people to use it?

Reply:
It's more of `protected` where we want the sub-classes to use it but not MigrationRunner or something else. What do you think? Shall I remove the `_` (But then it kind of becomes part of the Public API)?

Reply:
fine for me you can keep it